README.md

```diff
@@ -1 +1,31 @@
-# dmnlint-pack-config
\ No newline at end of file
+# dmnlint-pack-config
+
+[![CI](https://github.com/bpmn-io/dmnlint-pack-config/workflows/CI/badge.svg)](https://github.com/bpmn-io/dmnlint-pack-config/actions?query=workflow%3ACI)
```

Comment:
That doesn't work yet as there is no GitHub action available.

Can you maybe add one [similar to this one](https://github.com/nikku/bpmnlint-pack-config/tree/master/.github/workflows)?

Reply:
This is my first attempt at CI configurations - i've copied the configuration from [nikku/bpmnlint-pack-config](https://github.com/nikku/bpmnlint-pack-config/tree/master/.github/workflows). 

[feat(project): add CI configuration file](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/90b944637a42eafd4d365701e84058d66f680d0c)
[90b9446](https://github.com/bpmn-io/dmnlint-pack-config/pull/1/commits/90b944637a42eafd4d365701e84058d66f680d0c).